OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -269,11 +269,12 @@ public string VoicePhraseForOrder(Actor self, Order order)
 		public bool HasSpace(int weight) { return totalWeight + reservedWeight + weight <= Info.MaxWeight; }
 		public bool IsEmpty(Actor self) { return cargo.Count == 0; }
 
-		public Actor Peek(Actor self) { return cargo.Peek(); }
```

Comment:
Please keep the `self` parameter here for consistency.

Reply:
will do